frontend/src/components/SamplesList.tsx

```diff
@@ -231,8 +233,8 @@ export const SamplesList: FunctionComponent<ISampleListProps> = ({
       <AutoSizer>
         {({ width }) => (
           <div
-            className="ag-theme-alpine"
-            style={{ height: height, width: width }}
+            className={`ag-theme-alpine ${styles.tableHeight}`}
+            style={{ width: width }}
```

Comment:
This is part of a minor change that sets the AG Grid table height to fill up the available space, removing the unused gap below the table.